api/src/routing.js

```diff
@@ -425,6 +426,8 @@ app.get('/purging', authorization.onlineUserPassThrough, purgedDocsController.in
 app.get('/purging/changes', authorization.onlineUserPassThrough, purgedDocsController.getPurgedDocs);
 app.get('/purging/checkpoint', authorization.onlineUserPassThrough, purgedDocsController.checkpoint);
 
+app.get('/api/v1/log-replication-limit', logDocsController.getReplicationLimitLog);
```

Comment:
I don't think we need to expose this endpoint for anything currently. Did you have a use in mind?

If it does need to be exposed then we need to add some more security around it, for example, can we make it admin only? Or at least make it so non-admin users can only look up info about themselves?

Reply:
I was thinking that we can probably have a page in Admin app to show the logs of users exceeding the limit, so the admin can take actions.
After logging, how is the admin going to see the logs?

Reply:
The primary way is via the monitoring API, so it should be exposed as part of the JSON blob it returns. This is used for automated graphing and alerting, so at the very least we should return a count of how many users are above the limit. This is just an extension to the existing API and it's the main focus of this issue.

However this doesn't help for debugging which users are misconfigured, and if the monitoring isn't set up then it would be good to show it somewhere else.

There are two optional extensions I can see to this issue that you can choose to do if you like...

1. Also add this endpoint which is only available for admins and shows all users over the limit with the latest recorded count. Admins are usually technically capable enough to query an endpoint as part of their investigations.
2. Update the users page in the admin app to show a warning icon for users that have hit this limit. This makes the misconfiguration more obvious to less technical users.